lib/src/connector/connection.cc

```diff
@@ -498,6 +503,19 @@ WS_Context_Ptr Connection::onTlsInit(WS_Connection_Handle hdl)
                                   boost::asio::ssl::context::file_format::pem);
         ctx->load_verify_file(client_metadata_.ca);
 
+        // CODEREVIEW: Is this always safe? This is the pattern used for proxy...
```

Comment:
This entire method is already wrapped in a try/catch for std::exception, so we should be fine.

I also don't expect it to ever fail, since our default is set to the empty string.